src/js/global/rocket-insights.js

```diff
@@ -113,7 +115,10 @@ module.exports = (function () {
 		window.wp.apiFetch({
 			path: '/wp-rocket/v1/rocket-insights/pages/',
 			method: 'POST',
-			data: { page_url: url },
+			data: { 
+				page_url: url,
+				source: source || 'post type listing'
```

Comment:
We already do this check in line 43
```suggestion
				source: source
```

Reply:
Fixed in [20e60](https://github.com/wp-media/wp-rocket/commit/20e603374) - Removed redundant `|| 'post type listing'` fallback in API call since source is already determined.